full_snapshot.proto

```diff
@@ -479,6 +479,15 @@ message RelationStatistic {
 
   int32 frozenxid_age = 30; // The age of frozen xact ID for this relation (relative to latest stable xid)
   int32 minmxid_age = 31; // The age of minimum multixact ID for this relation (relative to latest stable mxid)
+
+  // For infrequently updated stats
```

Comment:
I don't like my English skill here, very much looking forward to some wording improvement reviews

Reply:
As a non-native speaker myself, I think this is acceptable English - you could copy some of the pg_class wording here for clarity, e.g.:

```
// Statistics that are infrequently updated (e.g. by VACUUM, ANALYZE, and a few DDL commands)
```